test/jdk/sun/net/www/http/KeepAliveStreamCleaner/KeepAliveStreamCleanerTestDriver.java

```diff
@@ -23,6 +23,8 @@
 
 /*
  * @test
+ * @bug 8255124 8255583
+ * @summary Tests that KeepAliveStreamCleaner run does not throw an IllegalMonitorState Exception.
```

Comment:
Thanks for the summary!
`@bug` should only list `8255124` ; We typically don't list issues that affects the tests only. We only list those that affect the JDK source code and which can be verified by running the test.

Reply:
Understood, I removed the additional id and now only 8255124 is present.